astroplan/core.py

```diff
@@ -359,6 +415,22 @@ def altaz(self, time, target=None, obswl=None):
             If ``target`` is `None`, returns `~astropy.coordinates.AltAz` frame.
             If ``target`` is not `None`, returns the ``target`` transformed to
             the `~astropy.coordinates.AltAz` frame.
+
+        Examples
+        --------
+        Create an instance of the `~astropy.coordinates.AltAz` frame for an
+        observer at Apache Point Observatory at a particular time; and
+        transform that
+
+        >>> from astroplan import Observer
+        >>> from astropy.time import Time
+        >>> from astropy.coordinates import SkyCoord
+        >>> apo = Observer.at_site("APO")
```

Comment:
"APO" is cryptic ... no idea where that is. Can we always use "Subaru" or other well-known sites?